### ISSUE TITLE
$review_author getting stripped from Review - Rich Snippet Error "no author"

### ISSUE THREAD

**Issue by laserjobs** [Automattic/amp-wp#267]
```
$box .= '<div>Reviewed by: <span class="reviewer author byline vcard hcard">';
					$box .= '<span class="author me fn" itemprop="author">' . $review_author .'</span></span></div>';
```

Renders Reviewed by: 
with no author

Tried the following but no author is rendered either
`$box .= '<meta content="' . $review_author .'" itemprop="author">';`

Here is the code that is not working
https://plugins.trac.wordpress.org/browser/author-hreview/trunk/include/review_template.php

**Comment by laserjobs** [comment]
Does anyone have any idea of why this would be getting stripped?  I ended up hard coding an author like this until I can figure out a solution:
`$box .= '<meta content="Author Name" itemprop="author">';

**Comment by authorhreview** [comment]
We've solved this in Author hReview version 0.0.9.8
